metadata-ingestion/src/datahub/ingestion/source/gc/soft_deleted_entity_cleanup.py

```diff
@@ -203,11 +205,10 @@ def _process_futures(self, futures: Dict[Future, str]) -> Dict[Future, str]:
         for future in done:
             self._print_report()
             if future.exception():
-                logger.error(
-                    f"Failed to delete entity {futures[future]}: {future.exception()}"
-                )
                 self.report.failure(
-                    f"Failed to delete entity {futures[future]}",
+                    title="Failed to delete entity",
+                    message=futures[future],
```

Comment:
Given we have the futures in the context, we may better keep a textual message - highly opinionated

```suggestion
                    message="Failed to delete entity",
```

Reply:
done